spec.bs

```diff
@@ -2295,6 +2362,32 @@ An interest group ad is a [=struct=] with the following items:
 
 </dl>
 
+<h3 dfn-type=dfn>Currency tag</h3>
+A currency tag is a [=string=] containing exactly 3 upper-case ASCII letters, or null. The null value is used to denote that the currency is unspecified.

```

Comment:
dumb question: if there an RFC or other place that defines these 3 letter tags? Is it https://www.iso.org/iso-4217-currency-codes.html? Something else?

Reply:
ISO 4217 was the intent, but maybe the industry could agree on some other values, too.  I hear they like doing things in cents, for example?

Reply:
OK, eventually we should link out to something... or invent a new thing (sure, why not). But that can be a follow up issue - I'll file a spec issue. 💸

Reply:
Filed https://github.com/WICG/turtledove/issues/634